mobilenet/mobilenetv3/mobilenet_v3.py

```diff
@@ -4,7 +4,6 @@
 import argparse
 
 import numpy as np
-import pycuda.autoinit
```

Comment:
This line can't be deleted. Add a noqa marker.

```
import pycuda.autoinit  # noqa: F401
```

Reply:
Apologies for the confusion earlier. Thank you for your patience!